client/src/TitleScreen/Components/Credits.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="TitleScreen.css">
+    </head>
+    <body>
+        <div class="Credits-Heading">Credits</div>
```

Comment:
We only need the contents of the body without the other HTML boilerplate like the actual `<body>` tag or the `<!DOCTYPE html>`

Check the other components for refrence.